### PR TITLE
Rearranging the collaborating episodes

### LINKED ISSUE #107: switch order of collaboration episodes?
We should revisit the order of the two collaboration skills episodes. Would the make more sense as "Team" first, then "Newcomers"?

### PR DESCRIPTION
**Pull request by sstevens2**
Trying this out in next week's workshop.  Setting up the PR, though we might not get it merged prior to the workshop next week.

Fixes #107 

cc: @anenadic

### CODE REVIEW

**Comment by sstevens2** [comment]
Wise or not, I bundled a few small changes to the lesson, typos other lang changes, with this PR to switch the order of the collaborating episodes.  I understand if we need to discuss any of those more throughly.  If it holds up this merge, I'll split it into multiple PRs as needed.
Also note, I didn't review as closely the part of collaborating with your team that Toby is teaching this time around.

**Review by anenadic — APPROVED** [review]
I have added some minor changes (mainly moved some of the intro sentences from the 'newcomers' to the 'collaborating with your team' episode). Happy to merge this if you agree with my suggestions @sstevens2.

**Comment by anenadic** [comment]
I have not added another commit to wedge in the "universal design" and cut-curb effect principle when working with newcomers - this is now about disability but increased accessibility. When we design our project with new collaborators in mind we are making it easier for everyone (to find info, collaborate, contribute). But it may be too much so wonder what others think @sstevens2 @tobyhodges ?

File: episodes/26-external.md [hunk 1]
```diff
@@ -171,6 +170,7 @@ but you might consider using it to do one or more of the following:
 
 - *"If a project doesn’t make a good first impression, newcomers may wait a long time before giving it a second chance" - Karl Fogel, the author of ["Producing Open Source Software: How to Run a Successful Free Software Project"](https://www.oreilly.com/library/view/producing-open-source/0596007590/)*.
 - You should make an active effort to attract potential collaborators and try to make them all feel welcome and included. [The Carpentries Help Wanted page](https://carpentries.org/help-wanted-issues/) and featuring in [the Incubator Lesson Spotlight](https://docs.carpentries.org/topic_folders/lesson_development/spotlight.html) can boost the visibility of your lesson. Creating the appropriate documentation and using GitHub features such as labels, and issue/pull request templates will help lower the barriers for contributions to your project.
+- When you design your lesson with new contributors and increased accessibility in mind, you make things better for everyone in the process
```
Comment (sstevens2) [inline episodes/26-external.md:173]:
```suggestion
- When you design your lesson with new contributors and increased accessibility in mind, you make things better for everyone in the process.
```

**Pull request merged**